config/initializers/carrierwave.rb

```diff
@@ -0,0 +1,36 @@
+# TODO: what happens if the ENVs are missing?
+# TODO: is ENV safe way to handle these credentials?
```

Comment:
ENV vars is a good way to deal with these credentials.

We should prolly add a sample `.env` file called `dot_env` and setup instructions in the README.
And the `dot_env` gem.